_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,257 @@
+
+<script>
+  const host = document.querySelector("#host");
+  const root = host.attachShadow({ mode: "open" });
+
+  root.innerHTML = `
+    <slot></slot>
+    <div role="listitem">List item 1</div>
+  `;
+</script>
+```
+
+### Failed
+
+#### Failed Example 1
+
+No context role.
+
+```html
+<div role="listitem"></div>
+```
+
+#### Failed Example 2
+
+Wrong context role.
+
+```html
+<div role="tablist">
+    <div role="listitem"></div>
+</div>
+```
+
+#### Failed Example 3
+
+Element not contained within its required context role.
+
+```html
+<div role="list"></div>
+    <div role="listitem"></div>
+```
+
+#### Failed Example 4
+
+Element with role `listitem` has a closer ancestor, that is included in the accessibility tree, than the role `list` that should have been its context role.
+
+```html
+<div role="list">
+    <div aria-label="menu">
+         <div role="listitem"></div>
+    </div>
+</div>
+```
+
+**Note:** This test case follows the definition of [owned by](#owned-by) used in this rule. If implemented differently, this definition could cause differences in outcome for this test case. 
+
+
+#### Failed Example 5
+
+Element with role `listitem` has a closer ancestor, that is included in the accessibility tree, than the role `list` that should have been its context role.
+
+```html
+<div role="list">
+  <div role="tabpanel">
+    <div role="listitem"></div>
+   </div>
+</div>
+```
+
+#### Failed Example 6
+
+The element with the semantic role of `listitem` is [owned by](#owned-by) the first element that references it element through `aria-owns`, which results in the wrong context role.
+
+```html
+<div role="tabpanel" aria-owns="id1">
+  <div role="list" aria-owns="id1">
+    <div id="id1" role="listitem"></div>
+   </div>
+</div>
+```
+
+**Note:** This test case follows the definition of [owned by](#owned-by) used in this rule. If implemented differently, this definition could cause differences in outcome for this test case.
+
+#### Failed Example 7
+
+Since explicit ownership cannot cross shadow boundaries, the element with the semantic role of `listitem` does not have a context role.
+
+```html
+<div role="list" aria-owns="item">
+</div>
+
+<div id="host"></div>
+
+<script>
+  const host = document.querySelector("#host");
+  const root = host.attachShadow({ mode: "open" });
+
+  root.innerHTML = `
+    <div id="item" role="listitem">List item 1</div>
+  `;
+</script>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+Element does not have an explicit semantic role.
+
+```html
+<ul></ul>
+```
+
+#### Inapplicable example 2
+
+Element is not exposed to assistive technologies.
+
+```html
+<div role="tab" aria-hidden="true"></div>
+```
+
+#### Inapplicable Example 3
+
+Role does not have any required context roles listed in WAI-ARIA spec.
+
+```html
+<div role="radio"></div>
+```
+
+#### Inapplicable Example 4
+
+Element is not exposed to assistive technologies, since decendant has `aria-hidden` attribute with value set to `true`.
```

Comment:
decendant > descendant

Reply:
It's not the "descendant" but the "parent" that has the aria-hidden property no?